app/json/data.json

```diff
@@ -0,0 +1,17 @@
+[{
+    "date": "10-03-2013",
```

Comment:
@iammidhu @Priyaqb ,
Why do we need this? We just need the categories and the rest should be added by the user.